__tests__/app.test.js

```diff
@@ -46,10 +46,49 @@ describe('GET /api', () => {
     });
 });
 
+describe('GET: /api/articles/:article_id', () => {
+    test('should return the article by id with the correct properties and 200 status', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toMatchObject({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String)
+                })
+            expect(body.article.article_id).toBe(1)
+            expect(body.article.topic).toBe('mitch')
+        })
+    });
+    test('should return a 400 Bad Request if given an invalid id', () => {
+        return request(app)
+        .get('/api/articles/invalid')
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe('Bad Request')
```

Comment:
The error message could be a bit more specific. `invalid input` instead of `Bad Request`